backend/data/seedFlowers.js

```diff
@@ -0,0 +1,33 @@
+import { Flower } from '../models/FlowerModel.js';
```

Comment:
Refers to another comment in the` FlowerModel.js`:  To remain coherent witht he syntax used in other models, is it possible to change `Flower` into `FlowerModel`?

Reply:
Name is updated in all files